treeherder/etl/classification_loader.py

```diff
@@ -88,3 +100,46 @@ def get_push(self, task_route):
             raise
 
         return push
+
+    def autoclassify_failures(self, failures):
+        try:
+            autoclassified_intermittent = FailureClassification.objects.get(
```

Comment:
Move that in `process` so you provide the list of  failures and the matching `FailureClassification` here (we may need to support different cases later on and it seems cleaner)